### PR TITLE
Upgrade Dependencies

### PR DESCRIPTION
**Pull request by azizhk**

### CODE REVIEW

File: src/components/Input.tsx [hunk 1]
```diff
@@ -119,16 +120,28 @@ class Input extends React.PureComponent<InputProps, InputState> {
...
+            aria-label={placeholder}
+            value={value || ""}
+            onChange={this.handleChange}
+            disabled={disabled}
+            readOnly={readOnly}
+          />
+        ) : (
+          <input
+            className={_inputClassName}
+            type={type}
```
[the remark this answers is not in the document]
Comment (azizhk) [reply src/components/Input.tsx]: Or we can avoid JSX and use React.createElement
Something like https://github.com/ritz078/pebble/pull/141/files#diff-288d98c45cab569122e391278589e7dbR122

[the remark this answers is not in the document]
Comment (ritz078) [reply src/components/Input.tsx]: we should prefer type correctness if the overhead is not too much.

[the remark this answers is not in the document]
Comment (ritz078) [reply src/components/Input.tsx]: can't the props be extracted out as a common object though?

[the remark this answers is not in the document]
Comment (azizhk) [reply src/components/Input.tsx]: Done.

**Pull request merged**